src/main/java/frc/trigon/robot/subsystems/collector/CollectorIO.java

```diff
@@ -0,0 +1,30 @@
+package frc.trigon.robot.subsystems.collector;
+
+import frc.trigon.robot.constants.RobotConstants;
+import frc.trigon.robot.subsystems.collector.kablamacollector.KablamaCollectorIO;
+import org.littletonrobotics.junction.AutoLog;
+
+public class CollectorIO {
+    static CollectorIO generateIO(){
+        if (RobotConstants.IS_REPLAY)
+            return new CollectorIO();
+        if (RobotConstants.ROBOT_TYPE == RobotConstants.RobotType.KABLAMA)
+            return new KablamaCollectorIO();
+        return new SimulationCollectorIO();
+    }
+    protected void updateInputs(CollectorInputsAutoLogged inputs) {
+    }
+
+    protected void setPower(double power) {
+    }
+
+    protected void stop() {
+    }
+
+    @AutoLog
+    protected static class CollectorInputs {
+        public double
+                motorCurrent = 0,
+                motorVoltage = 0;
```

Comment:
For these, the `public double` isn't redundant.

Reply:
okay, but why though